### ISSUE TITLE
write more comments before lines in example notebooks

### ISSUE THREAD

**Issue by pmcdonal** [cosmodesi/desilike#9]
E.g., in fisher_planck.ipynb, there is a line
fisher_clik = Fisher(likelihood_clik)
which takes quite a bit of time and looks like it might be doing derivatives, but at the end there is, e.g., no "precision" set so I guess not. Then the line 
precision_clik = fisher_clik()
takes a longer time (printing nothing), where now I understand it must be doing the real calculation.
(I took the change to 3.044 out in case this was  forcing recalculation, before realizing I could check that no "precision" was set before)

So I am curious what the first line was doing, but the more general comment is that a lot more comments in the notebooks would help really understand what is going on. (I'd like to just read through code path/step through in debugger... but I haven't had great luck with that)

**Comment by johannesulf** [comment]
@pmcdonal Agreed! The notebooks should contain enough information that users don't have to guess what each command does under the hood. (Ideally, the API should be clear enough that one can infer functionality even without comments.) I'll keep this open until we have a complete set of easy-to-follow notebooks covering most use cases.
